scripts/templates/EmptyReadme.mustache

```diff
@@ -0,0 +1,17 @@
+# [Office UI Fabric React {{friendlyPackageName}}](http://dev.microsoft.com/fabric)
+
+##### {{friendlyPackageName}} components for Office UI Fabric React
+
+These are not production ready components and __should never be used in product__. This {{friendlyPackageName}} space is useful for testing new components whose APIs might change before final release.
+
+To import {{friendlyPackageName}} components:
+
+```js
+import { ComponentName } from '@uifabric/{{packageName}}/lib/ComponentName';
+```
+
+Once the {{friendlyPackageName}} component graduates to a production release, the component will be available at:
+
+```js
+import { ComponentName } from 'office-ui-fabric-react/lib/ComponentName';
```

Comment:
Cliff and Ken have mentioned that we should now be using package level imports, which I think would affect lines 10 and 16.